install/operator/pkg/syndesis/capabilities/capabilities.go

```diff
@@ -37,15 +37,15 @@ type RequiredApiSpec struct {
 	routes                    string
 	imagestreams              string
 	oauthclientauthorizations string
-	catalogsources            string
+	packagemanifests          string
```

Comment:
I'm assuming this is part of this PR?

Reply:
Looks like it needs to be.

Reply:
Yes, I replaced `catalogsources` for `packagemanifests` to test if openshift can install from OLM. At least in my openshift 3.11, `catalogsources` does exist but lacks OLM operator and other OLM objetcs. 
Therefore, I found that `packagemanifests` better represents a CR to enable operators to install from operatorhub.